content/docs/web-components.md

```diff
@@ -20,14 +20,14 @@ class HelloMessage extends React.Component {
 }
 ```
 
-> Note:
+> Pamiętaj:
 >
-> Web Components often expose an imperative API. For instance, a `video` Web Component might expose `play()` and `pause()` functions. To access the imperative APIs of a Web Component, you will need to use a ref to interact with the DOM node directly. If you are using third-party Web Components, the best solution is to write a React component that behaves as a wrapper for your Web Component.
+> Komponenty sieciowe często udostępniają imperatywne API. Na przykład komponent sieciowy `video` może udostępniac funkcje `play()` i `pause()`. Aby skorzystać z imperatywnego API komponentu sieciowego, możesz potrzebować użyć referencji, aby bezpośrednio nawiązać kontakt z węzłem DOMu. Jeśli korzystasz z komponentów sieciowych innych osób, najlepszym rozwiązanie będzie utworzyć komponent React służący jako nakładka dla komponentu sieciowego.
 >
-> Events emitted by a Web Component may not properly propagate through a React render tree.
-> You will need to manually attach event handlers to handle these events within your React components.
+> Wydarzenia emitowane przez komponent sieciowy mogą nie właściwie propagować się przez drzewo renderowania React.
+> Możesz być zmuszony ręcznie załączyć funkcję obsługującą wydarzenie, aby zajmować się tym wydarzeniem w ramach komponentu React.
 
-One common confusion is that Web Components use "class" instead of "className".
+Mylące potrafi być, że komponenty sieciowe używają parametru "class" zamiast "className".
```

Comment:
```suggestion
Często mylącym aspektem komponentów sieciowych jest fakt, iż korzystają z atrybutu "class" zamiast "className".
```

Reply:
Pozostawiłem `Mylące potrafi być`, bo jest krótsze i bardziej jasne. Poza tym `Często mylącym` źle brzmi.

Reply:
`Mylące potrafi być, że` moim zdaniem brzmi znacznie gorzej, tak jakby to był zbitek słów, a nie zdanie.

Reply:
Jeśli już iść tą droga, to raczej zapis `Często mylonym aspektem` byłby prawidłowy. Trudno, żeby coś było często mylące.